index.html

```diff
@@ -2228,27 +2359,55 @@ <h4 class="modal-title">Help</h4>
 
                 board.init(5, "white", true);
             }
+            
+            function loadSettings() {
+              // load the setting for wall orientation.
+              if(localStorage.getItem('diagonal_walls')==='true') {
+                document.getElementById('wall-orientation').checked = true;
+              }
+              
+              // load the setting for antialiasing.
+              if(localStorage.getItem('antialiasing_mode')==='true') {
+                document.getElementById('antialiasing-checkbox').checked = true;
+                antialiasing_mode = true;
+              }
+              
+              // load the setting for automatically opening and closing the chat on window resize.
+              if(localStorage.getItem('auto_chat')==='false') {
+                document.getElementById('autoopen-chat-checkbox').checked = false;
+              }
+              
+              // load the setting for piece size.
+              if(!(localStorage.getItem('piece_size')===null)) {
+                piece_size = parseInt(localStorage.getItem('piece_size'));
+                document.getElementById('piece-size-display').innerHTML = piece_size;
+                document.getElementById('piece-size-slider').value = piece_size;
+              }
+            }
 
             function onWindowResize() {
-                canvas.width = window.innerWidth;
-                canvas.height = window.innerHeight;
+              canvas.width = window.innerWidth;
+              canvas.height = window.innerHeight;
 
-                renderer.setSize(canvas.width, canvas.height);
+              renderer.setSize(canvas.width, canvas.height);
 
-                camera.aspect = canvas.width / canvas.height;
-                camera.updateProjectionMatrix();
+              camera.aspect = canvas.width / canvas.height;
+              camera.updateProjectionMatrix();
 
-                $('#chat').offset({ top: $('nav').height() + 5 });
-                $('#chat-toggle-button').offset({ top: $('nav').height() + 7 });
-                $('#chat').height(window.innerHeight - $('nav').height() - 85);
+              $('#chat').offset({ top: $('nav').height() + 5 });
+              $('#chat-toggle-button').offset({ top: $('nav').height() + 7 });
+              $('#chat').height(window.innerHeight - $('nav').height() - 85);
 
+              if (!(localStorage.getItem('auto_chat')==='false'))
```

Comment:
=== true won't work, because the localStorage is not required to contain 'auto_chat'.
But !== can certainly be employed.